lib/strings.md

```diff
@@ -5,20 +5,26 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A string is a set (sorry, not trying to use that in an official sense) or sequence of characters.  Usually strings are a sequence of unicode characters that represent characters which themselves represent human language (http://www.tutorialspoint.com/ruby/ruby_strings.htm)
+So, 'This is a string' is a string of a sequence of characters.
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+Strings can be in single or double quotes.  So, collected elements in quotes are usually examples of strings.  Here is an instance of a string: example = "This is a string" , and another example is: statement = 'I am not a dog.'  Sequences of characters without quotation marks would not be strings, but could be name for the string for instance, or a key or value, for instance.
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+To retrieve the 6th character of the string, you would just put that character in brackets next to the string, like this: example[6], which in my example, above, would get "s" as a result.  In the string like "Ada Lovelace", we would get "v".  The 8th character would give us 'l' as a result. if we tried to get the 99th character, or a character that was not part of the string (because it is not in the length of that string, in this case), we would get nil as a return.
```

Comment:
If you're thinking of the `"A"` in `"Ada Lovelace"` as the 0th character, this answer makes sense to me!